rsts/howto/managing_customizable_resources.rst

```diff
@@ -57,39 +68,55 @@ In the absence of custom override values, templateData from the `flyteadmin conf
 
 Note that these settings can only take on domain, or a project and domain specificity. Since Flyte has not tied in the notion of a workflow or a launch plan to any Kubernetes constructs, specifying a workflow or launch plan name doesn't make any sense.
 
+Running the following, will make it so that when Admin fills in cluster resource templates, the K8s namespace ``flyteexamples-development`` will have a resource quota of 1000 CPU cores and 5TB of memory.
+
+.. code-block:: bash
 
-Command
--------
-Running the following, will make it so that when Admin fills in cluster resource templates, the K8s namespace ``projectname-staging`` will have a resource quota of 1000 CPU cores and 5TB of memory.
+    flyte-cli -h localhost:30081 -p flyteexamples -d development update-cluster-resource-attributes  \
+    --attributes projectQuotaCpu 1000 --attributes projectQuotaMemory 5000Gi
 
-.. code-block:: console
 
-    curl --request PUT 'https://flyte.company.net/api/v1/project_domain_attributes/projectname/staging' --header 'Content-Type: application/json' --data-raw '{"attributes":{"matchingAttributes":{"clusterResourceAttributes":{"attributes":{"projectQuotaCpu": "1000", "projectQuotaMemory": "5000Gi"}}}}}'
+These values will in turn be used to fill in the template fields
+`in the file <https://github.com/flyteorg/flyte/blob/master/kustomize/base/single_cluster/headless/config/clusterresource-templates/ab_project-resource-quota.yaml>`__
```

Comment:
you can actually do `.. remoteinclude::`

Reply:
okay, done